alembic/versions/373ef4569b93_correlations_table.py

```diff
@@ -0,0 +1,71 @@
+"""correlations table
+
+Revision ID: 373ef4569b93
+Revises: 335c2bfd99a6
+Create Date: 2016-04-25 14:11:28.373859
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+from socorrolib.lib import jsontype
+
+# revision identifiers, used by Alembic.
+revision = '373ef4569b93'
+down_revision = '335c2bfd99a6'
+
+
+def upgrade():
+    op.create_table(
```

Comment:
It's important that migrations continue to work over time as the rest of the codebase changes. Given that, the migration explicitly can't depend on the rest of the codebase for information or functionality. The same is true for Django migrations and South migrations.